src/util/filters.js

```diff
@@ -0,0 +1,43 @@
+import {expandToString} from "./formatValue";
+import {Range} from "../constants/filters";
+
+export const parseFilterValue = (value, isDai = false) => {
+  value = parseFloat(value);
+
+  if (isDai) {
+    value = expandToString(value * 1e18);
+  }
+
+  return value;
+}
+
+export const parseRangeQuery = (type, options, values, isDai) => {
+  switch (type) {
+    case Range.GTE:
+    case Range.LTE:
+      return {
+        [options.field]: {
+          [type]: parseFilterValue(values[type], isDai)
+        }
+      };
+
+    case Range.BETWEEN:
+      return {
+        [options.field]: {
+          [Range.LTE]: parseFilterValue(values[Range.LTE], isDai),
+          [Range.GTE]: parseFilterValue(values[Range.GTE], isDai),
+        }
+      }
+  }
+}
+
+export const parseCheckboxQuery = (options, values) => {
+  let query = {};
```

Comment:
Nit: this is just a style nit but can be nice to keep things immutable and only use reassignments when you have to. Here for example you could use the `Array.reduce` method. eg:

```js
const query = values.reduce((acc, value) => { ... }, {});
```